src/utilities/createRouter.ts

```diff
@@ -83,9 +77,32 @@ export function createRouter<const T extends Routes>(routes: T, options: RouterO
     })
 
     if (!success) {
-      return
+      done()
+    }
+  }
+
+  const onAfterLocationUpdate: AfterLocationUpdate = async (url) => {
+    const to = getResolvedRouteForUrl(routerRoutes, url)
+    // const from = isRejectionRoute(route) ? null : route
+
+    if (!to) {
+      return reject('NotFound')
     }
 
+    // const success = await executeRouteHooks({
+    //   hooks: [
+    //     ...hooks.after,
+    //     ...getRouteHooks(to, from, 'after'),
+    //   ],
+    //   to,
+    //   from,
+    //   onRouteHookError,
+    // })
+
+    // if (!success) {
+    //   return
+    // }
```

Comment:
Commenting this out because `getRouteHooks` with `after` timing isn't implemented yet.